src/util.rs

```diff
@@ -230,49 +229,67 @@ pub fn print_blocks(
         }
 
         // Get the final JSON from all the widgets for this block
-        let rendered_widgets = widgets
+        let mut rendered_widgets = widgets
             .iter()
             .map(|widget| {
+                let mut w_json: serde_json::Value = widget.get_rendered().to_owned();
                 if alternator {
                     // Apply tint for all widgets of every second block
-                    let mut w_json: serde_json::Value = widget.get_rendered().to_owned();
                     *w_json.get_mut("background").unwrap() = json!(add_colors(
-                        &w_json["background"].to_string()[1..],
-                        &config.theme.alternating_tint_bg
+                        w_json["background"].as_str(),
+                        config.theme.alternating_tint_bg.as_ref()
```

Comment:
Try changing this to `.as_deref()`

Reply:
Neat, i did not know this was a possibility :)